src/redux/reducer.ts

```diff
@@ -428,7 +473,7 @@ type RootNetworkUpdatedNotification = {
 
 type RootNetworkUpdateFailedNotification = {
     type: NotificationType.ROOT_NETWORKS_UPDATE_FAILED;
-    eventData: RootNetworkModifiedEventData;
+    eventData: RootNetworkUpdateFailedEventData;
```

Comment:
I think all we need is a type StudyUpdated in which we have eventData : un union type od all XXXEventData